lib/matplotlib/colors.py

```diff
@@ -891,12 +891,12 @@ def process_value(value):
                 # (list, tuple, deque, ndarray, Series, ...)
                 result = result.copy()
             elif result.dtype.itemsize > 2:
-                result = result.astype(np.float)
+                result = result.astype(float)
             else:
                 result = result.astype(np.float32)
         else:
             is_scalar = True
-            result = ma.array([value]).astype(np.float)
+            result = ma.array([value]).astype(float)
```

Comment:
If you are standardizing on `np.ma`, here is another opportunity for it.  I used to use `ma`, but now I always use `np.ma`.